### ISSUE TITLE
HA Entities disappear or gets automatically rearranged in Android 11 Control Center

### ISSUE THREAD

**Issue by maretodoric** [home-assistant/android#1616]
**Home Assistant Android version:**
2021.6.2-full

**Android version:**
11, RP1A.200720.011 (MIUI 15.5.1)

**Phone model:**
Xiaomi Redmi Note 8 Pro

**Home Assistant version:**
Core: core-2021.7.3
Supervisor: supervisor-2021.06.8
OS: Home Assistant OS 6.1

**Last working Home Assistant release (if known):**
N/A

**Description of problem:**
**1st scenario:**
- In Android 11, Control Center was introduced where you will dropdown the notification bar (control center part) and there are Smart Devices there.
- In settings, you choose the smart home app that you will be using (Home, Mi Home, Home Assistant, etc...)
- When home assistant is selected, you can later rearrange the devices in Control Center
- After some time (unknown) rearranged icons will disappear and will revert to default, when i try to arrange them again, no devices will show up so i cannot remove the current devices (which are useless automations, helpers and scripts) to add switches that I'm actually using

**2nd scenario:**- In Android 11, Control Center was introduced where you will dropdown the notification bar (control center part) and there are Smart Devices there.
- In settings, you choose the smart home app that you will be using (Home, Mi Home, Home Assistant, etc...)
- When home assistant is selected, you can later rearrange the devices in Control Center
- After some time (unknown) rearranged icons will disappear and will revert to default. At that point i have to edit the items again, removing the unused/unneeded and adding the useful switches and devices, which can take long time depending on size of HA, to find the right entity etc...

**Traceback (if applicable, to get the logs you may refer to: https://companion.home-assistant.io/docs/troubleshooting/faqs/#android-crash-logs):**
None

**Screenshot of problem:**
Failed to get screenshot when devices are missing, will do that once the issue reappears.

**Additional information:**
Not sure if this is something to be handled by Home Assistant or MIUI. If some logs are needed please let me know.

**Comment by dshokouhi** [comment]
This sounds like a OS bug as the app does not control the appearance or arrangement.  We only provide a list and the OS goes by that.

**Comment by maretodoric** [comment]
Though so myself but weird thing that contradicts that is that when i set Google home to be the smart home app it works fine.
So not really sure why they have a problem with HA

**Comment by dshokouhi** [comment]
try checking the logs in app configuration, thats the only thing that might give us a clue....open the app, reproduce the issue, then go back to the app and get the logs

this is the first report i have heard of this and I do not see the issue on my Pixel 4 XL

**Comment by maretodoric** [comment]
I'll try to replicate and get the log. It all seems very random.
Thanks for recommendation and for looking into this

**Comment by maretodoric** [comment]
Ok, i managed to replicate this. Every time.
- I will setup icons as i want them to
- I reboot the phone
- After reboot, they revert back to default

I've generated log files this way (for them to have as small footprint as possible):
I have rebooted the phone, then configured the icons. This is the log file when I've configured the order and entities i want in Control Center, before reboot: https://pastebin.com/s1HK6C0d

And then i rebooted and immediately checked icons in Control Center, they were defaulted, then i grabbed another log bundle, here:
https://pastebin.com/4gZiKeZa

I'm not really sure what to look for in the log files.
